modules/utxo-lib/src/bitgo/zcash/ZcashPsbt.ts

```diff
@@ -128,14 +129,30 @@ export class ZcashPsbt extends UtxoPsbt<ZcashTransaction<bigint>> {
   // transactions because zcash hashes the value directly. Thus, it is unnecessary to have
   // the previous transaction hash on the unspent.
   signInput(inputIndex: number, keyPair: Signer, sighashTypes?: number[]): this {
-    (this as any).__CACHE.__UNSAFE_SIGN_NONSEGWIT = true;
+    this.setUnsafeSignNonSegwit(true);
```

Comment:
Let's do this as a `with` now that we're making it cleaner.

Reply:
Is it depreciated? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/with